tests/unit_tests/test_geoip_service.py

```diff
@@ -0,0 +1,53 @@
+from aiohttp import web
+import gzip
+import os
+import random
+import string
+from unittest.mock import Mock
+
+import server.config
+from server.geoip_service import GeoIpService
+
+
+async def test_check_update(geoip_service):
+    path = '/tmp/fake_db.mmdb'
+    with open(path, 'a'):
+        os.utime(path, (0, 0))
+    server.config.GEO_IP_DATABASE_MAX_AGE_DAYS = 32
+    geoip_service.file_path = '/tmp/fake_db.mmdb'
```

Comment:
geoip_service.file_path = '/tmp/fake_db.mmdb' - copy paste kung-fu, how about to move all common configuration to fixure?

For different cases: how about to parametrize fixture via `request`? https://docs.pytest.org/en/latest/example/parametrize.html